src/config.ts

```diff
@@ -49,7 +71,7 @@ function getConfig() {
         asyncOutputDestination: configOptions.get("sendAsyncOutputTo") as string,
         customREPLCommandSnippets: configOptions.get("customREPLCommandSnippets", []),
         customREPLCommandSnippetsGlobal: configOptions.inspect("customREPLCommandSnippets").globalValue as customREPLCommandSnippet[],
```

Comment:
Are you thinking that the config format should stay so similar that we can use both regular VS Code settings and EDN files for this? If so maybe we can keep both global and workspace config in settings and merge the EDN configs. (See my other comment about existing configs.)

Reply:
Yes, after playing with it a bit more that makes sense to me.
The local .calva/config.edn would have the same calva settings as .vscode/settings.json. The something.jar/calva.export/config.edn would have dependency-specific settings like custom snippets.